inst/rmarkdown/templates/icesjms/resources/template.tex

```diff
@@ -0,0 +1,263 @@
+$if(colorlinks)$
+\PassOptionsToPackage{dvipsnames,svgnames*,x11names*}{xcolor}
+$endif$
+
+\documentclass[$if(fontsize)$$fontsize$,$else$12pt,$endif$$if(spacing)$$spacing$,$else$halfline,$endif$$if(papersize)$$papersize$,$else$a4paper,$endif$$for(classoptions)$$classoptions$$sep$,$endfor$]{ouparticle}
+
+% Packages I think are necessary for basic Rmarkdown functionality
```

Comment:
There could be packages that could be required by Pandoc to work correctly. **rmarkdown** won't itself require some packages I believe. 

The packages are listed in here: https://pandoc.org/MANUAL.html#creating-a-pdf
and can be optionally used dependending on what option is used. Best guess would be to compare with default template https://github.com/jgm/pandoc/blob/master/data/templates/default.latex

So I don't know the list exactly and if we want to check we need to remove and try with most feature we want activated. 

I trust you on this list.